components/automate-ui/src/app/entities/service-groups/service-groups.model.ts

```diff
@@ -1,11 +1,11 @@
 import { RollupServiceStatus, SortDirection  } from '../../types/types';
 
 export interface HealthSummary {
-  total: number;
-  ok: number;
-  warning: number;
-  critical: number;
-  unknown: number;
+  total?: number;
+  ok?: number;
+  warning?: number;
+  critical?: number;
+  unknown?: number;
```

Comment:
Generally I would stay away from optional fields in an interface. Instead you may want to just set the fields with some default values in the initial state.

Reply:
I can do that! Thanks for your feedback Matt.